### DISCUSSION TITLE
Migrating from docker compose v1 to v2

### DISCUSSION THREAD

**Discussion by maximmax42** [diced/zipline#300] [General]
Basically, compose v1 (`docker-compose` with the dash; a separate command) is gonna be deprecated by the end of June this year, so it's a good idea to migrate whole project to use compose v2 (`docker compose` with the space; a plugin for docker).
From my experience running zipline, there's nothing additional that needs to be configured, so you can just literally find and replace all instances of `docker-compose` with `docker compose` within the repo and the docs (except for docker-compose(.dev).yml files of course), but I can't say for sure it won't break something.

**Comment by Zyztem** [comment]
Not that I have seen, I'm using v2 compose and it works fine

**Comment by diced** [comment]
Docs and main repo (here) has been updated.
